docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
+
+The delegate must have a baking power of at least 6,000 tez to be allowed to bake.
+
+A delegate also participates in [governance](./governance) in proportion to their _voting power_.
+The voting power of a delegate is similar to the baking power except that no distinctions is made between staked and non-staked tez.
 
 Bakers must run at least one Tezos node and a baker daemon to go with it.
-These daemons must run at all times with a stable internet connection, because inactive bakers are automatically removed from the network.
+These daemons must run at all times with a stable power source and internet connection, because periods of inactivity for a baker entail losses of rewards, and eventually being marked as inactive and temporarily excluded from baking.
 
 ## Delegating to a baker
 
-If you don't have enough tez to become a baker or don't want to run a baking node, you can delegate tez to a baker, which makes you a _delegator_.
-The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but the tez that you delegate counts toward the amount of tez that the delegate has for baking purposes.
-Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they delegate.
+If you don't have enough tez to become a baker or don't want to run a baking node, you can choose a baker as your delegate, which makes you a _delegator_.
+The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but half of the tez that you delegate counts toward the baking power of the delegate. Also, delegated tez increase the voting power of your baker: a delegate's voting power is the sum of its own tez plus the tez delegated to it.
+
+In exchange, delegates may share some part of their rewards with you, in proportion to the amount of tez that you currently have.
+Check your delegate's conditions for distributing rewards.
+
+Delegating incurs no risk for the delegator: the delegate has no control over the tez and the delegator is not punished if the delegate misbehaves.
+
+## Staking with a baker
+
+If you want to get a bigger share of your delegate's rewards, you can temporarily stake any amount of tez from your account on behalf of your baker.
+All the tez you stake contribute to the baker's security deposit, and are locked under the same conditions as the baker's own staked tez, hence are slashed if the baker misbehaves.
+In exchange of agreeing to temporarily lock these tez and sharing the risks with your delegate, you get a bigger share of the delegate's rewards.
+
+Later on, you may choose to unstake part or all of the staked tez, that will be unlocked after a certain delay.
+
+To stake tez, you start by delegating to a baker's account, then you stake a certain amount of your tez to the baker.
```

Comment:
You start by choosing a baker as your delegate, if you haven't done so already, and then you decide how much you stake.

Reply:
It might be good to clarify that you can only stake with **one** baker.

Reply:
That is, a staker is a delegator that stakes tez with the chosen baker.

Reply:
Clarified.